README.md

```diff
@@ -39,13 +39,21 @@ curl -L https://github.com/redhat-developer/odo/raw/master/scripts/install.sh |
 #### macOS
 
 ```sh
-sudo curl -L  "https://github.com/redhat-developer/odo/releases/download/v0.0.6/odo-darwin-amd64.gz" | gzip -d > /usr/local/bin/odo; chmod +x /usr/local/bin/odo
+# Binary installation
+curl -L https://github.com/redhat-developer/odo/releases/download/v0.0.6/odo-darwin-amd64 -o odo && chmod +x odo && sudo mv odo /usr/local/bin/odo
```

Comment:
sorry, noticed this just now.
could we make this shorter?

`sudo curl -L https://github.com/redhat-developer/odo/releases/download/v0.0.6/odo-darwin-amd64 -o  /usr/local/bin/odo && chmod +x /usr/local/bin/odo`

Reply:
that works